Asgard.Console/ServiceMode.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Asgard.Communications;
+using Asgard.EngineControl;
+using Terminal.Gui;
+
+namespace Asgard.Console
+{
+    internal class ServiceMode : Window
+    {
+        private readonly EngineManager engineManager;
+        private readonly TextField cv;
+        private readonly TextField value;
+        private IEngineSession session;
+        public ServiceMode(ICbusMessenger cbusMessenger)
+        {
+            this.engineManager = new EngineManager(cbusMessenger);
+
+            this.Add(new Label() { Text = "CV: ", X = 0, Y = 0 });
```

Comment:
Probably worth having a routine that returns an enumeration of these controls that can be called from the constructor where an AddRange is done.

Reply:
I've tried a few different ways to refactor this out of the constructor, but not overly happy with any of them - needing to set the class level fields for the text fields doesn't help etc.  Since this is another view built the same way as the other ones in the application, I think I'd prefer to tackle this in a different branch and update the structure of all the different views in the whole app once a different approach is identified.

Reply:
That makes sense.